### PR TITLE
Add debug headers to proxied endpoints

### PR DESCRIPTION
**Pull request by schwers**

### CODE REVIEW

**Comment by schwers** [comment]
👓  @nramadas @curioussavage 

cc 👓  @jackniu1

**Comment by curioussavage** [comment]
🐟  for me after in person review

File: src/lib/proxiedApiOptions.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/**
+* Updates the default server side api options with usefull debugging info
```
Comment (nramadas) [inline src/lib/proxiedApiOptions.js]: nit, spacing on these

**Comment by curioussavage** [comment]
whoops looks like travis failed

File: src/lib/proxiedApiOptions.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+* @param   {object} apiOptions - The base api options to pass to node-api-client
+* @returns {object} The updated api options with useful debugging info attached
+*/
+
+import config from 'config';
+
+export default function proxiedApiOptions(ctx, apiOptions) {
+  const proxiedHeaders = {};
+
+ config.apiPassThroughHeaders.forEach(key => {
```
Comment (schwers) [inline src/lib/proxiedApiOptions.js]: yeah something weird happened with spacing in here

**Comment by schwers** [comment]
Just for a followup, part of the travis failure was bugs in tests from a previous commit, I've added a quick fixup commit for those

:haircut_man:

**Comment by curioussavage** [comment]
Fixups look good to me.

**Pull request merged**